Tests/Command/CleanCommandTest.php

```diff
@@ -0,0 +1,168 @@
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\OAuthServerBundle\Tests\Command;
+
+use FOS\OAuthServerBundle\Command\CleanCommand;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class CleanCommandTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var \FOS\OAuthServerBundle\Command\CleanCommand
+     */
+    private $command;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|\Symfony\Component\DependencyInjection\ContainerInterface
+     */
+    private $container;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp()
+    {
+        $command = new CleanCommand();
+
+        $application = new Application();
+        $application->add($command);
+
+        $this->container = $this->getMock('Symfony\Component\DependencyInjection\ContainerInterface');
+
+        $this->command = $application->find($command->getName());
+        $this->command->setContainer($this->container);
+    }
+
+    /**
+     * Delete expired tokens for classes that implement the TokenManagerInterface.
+     */
+    public function testItShouldRemoveExpiredTokensForTokenManagerInterfaces()
+    {
+        $expiredAccessTokens = 5;
+        $accessTokenManager = $this->getMock('FOS\OAuthServerBundle\Model\TokenManagerInterface');
+        $accessTokenManager
+            ->expects($this->once())
+            ->method('deleteExpired')
+            ->will($this->returnValue($expiredAccessTokens));
+
+        $expiredRefreshTokens = 3;
+        $refreshTokenManager = $this->getMock('FOS\OAuthServerBundle\Model\TokenManagerInterface');
+        $refreshTokenManager
+            ->expects($this->once())
+            ->method('deleteExpired')
+            ->will($this->returnValue($expiredRefreshTokens));
+
+        $expiredAuthCodes = 0;
+        $authCodeManager = $this->getMock('FOS\OAuthServerBundle\Model\TokenManagerInterface');
+        $authCodeManager
+            ->expects($this->once())
+            ->method('deleteExpired')
+            ->will($this->returnValue($expiredAuthCodes));
+
+        $containerMap = array(
+            'fos_oauth_server.access_token_manager' => $accessTokenManager,
+            'fos_oauth_server.refresh_token_manager' => $refreshTokenManager,
+            'fos_oauth_server.auth_code_manager' => $authCodeManager,
+        );
+
+        $this->container
+            ->expects($this->exactly(count($containerMap)))
+            ->method('get')
+            ->will($this->returnCallback(function ($argument) use ($containerMap) {
+                return $containerMap[$argument];
+            }));
+
+        $tester = new CommandTester($this->command);
+        $tester->execute(array('command' => $this->command->getName()));
+        $display = $tester->getDisplay();
+
+        $this->assertRegExp(sprintf('\'Removed %d items from %s storage.\'', $expiredAccessTokens, 'Access token'), $display);
```

Comment:
That would be much cleaner for these matches indeed. It was leftover code from the `assertNotRegExp` from the last test where I actually needed to use `\d+` for matching.